pkg/adapter/nimbus-k8tls/builder/builder.go

```diff
@@ -6,29 +6,46 @@ package builder
 import (
 	"context"
 	"fmt"
+	"os"
+	"strconv"
 	"strings"
 
+	"github.com/5GSEC/nimbus/api/v1alpha1"
+	"github.com/5GSEC/nimbus/pkg/adapter/idpool"
+	"github.com/5GSEC/nimbus/pkg/adapter/k8s"
 	batchv1 "k8s.io/api/batch/v1"
 	corev1 "k8s.io/api/core/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	utilruntime "k8s.io/apimachinery/pkg/util/runtime"
+	"sigs.k8s.io/controller-runtime/pkg/client"
 	"sigs.k8s.io/controller-runtime/pkg/log"
+)
 
-	"github.com/5GSEC/nimbus/api/v1alpha1"
-	"github.com/5GSEC/nimbus/pkg/adapter/idpool"
+var (
+	DefaultSchedule         = "@weekly"
+	backOffLimit            = int32(5)
+	scheme                  = runtime.NewScheme()
+	k8sClient               client.Client
 )
 
 var (
-	DefaultSchedule           = "@weekly"
-	backOffLimit              = int32(5)
-	hostPathDirectoryOrCreate = corev1.HostPathDirectoryOrCreate
+	NamespaceName string
 )
 
+func init() {
+	utilruntime.Must(corev1.AddToScheme(scheme))
+	utilruntime.Must(batchv1.AddToScheme(scheme))
+	utilruntime.Must(v1alpha1.AddToScheme(scheme))
+	k8sClient = k8s.NewOrDie(scheme)
+}
```

Comment:
Since manager already adds these resources to scheme, so why are we again adding these to the scheme?
https://github.com/5GSEC/nimbus/blob/4fd48a2b7276cd595a3adb1b22d78ff534a1f085/pkg/adapter/nimbus-k8tls/manager/manager.go#L37-L41